skimage/data/__init__.py

```diff
@@ -254,8 +255,14 @@ def _fetch(data_filename):
 
 def _init_pooch():
     os.makedirs(data_dir, exist_ok=True)
+    dest_path = osp.join(data_dir, 'README.txt')
     shutil.copy2(osp.join(skimage_distribution_dir, 'data', 'README.txt'),
-                 osp.join(data_dir, 'README.txt'))
+                 dest_path)
+    # In case the user installing/running from a read-only filesystem,
+    # we need to ensure README.txt has user-write permission when it is
+    # put into their cache directory.
+    current = stat.S_IMODE(os.lstat(dest_path).st_mode)
+    os.chmod(dest_path, current | stat.S_IWUSR)
```

Comment:
Another option would be to check if the `dest_path` already exists, and only copy it if it doesn't.

Unless there's a particular reason why every `import skimage` needs to result in copying the `README.txt` file?

Reply:
I agree, but this leaves the ability to update the `README.txt` ... then again, if the cache directory has the version in it, then this shouldn't happen, I just now realize.

Reply:
I would like to hear from one of the scikit-image maintainers ... if there's a reason to recopy, we can leave this, or I'd be happy to change it to a check if the file exists. (Which is probably more efficient, although it's a tiny file.)

Reply:
it might just be better to read the file line by line, and just overwrite the file. The README.txt file in the data cache directory can be assume to be owned by us.

Reading and writing files with the default settings enables system admins to choose good defaults for their security model.

Reply:
Is there a reason it has to be overwritten, though?  Can't we just check that it exists and not copy/overwrite if it's there? I'm of a mind that would be the most reliable and fastest.

Reply:
I'm worried that the text won't get updated if we change the language.

I think you can skip the copy if the file exists for now. Please justify your decision in a comment in the code.

I'm rather agnostic for this small file.

Do we copy in any other files with `shutil`?

Reply:
I have updated to only copy if it isn't there, and added an explanation.  In general, I don't think this is a problem because the cache directory contains the scikit-image version number, and I can't see a path that README.txt would change within a given version/installation without shenanigans.

As for other copies, yes, the other data files are all copied in.  However, they have their hash checked so will not be re-copied if their hashes match.  It is possible that somebody could make a file writeable, modify it, and put it back to read only and then one of these `copy2` actions would fail, but that is hard to protect against.  As for the files themselves changing from version to version, that is possible but they would be put into a different cache directory bypassing the overwrite problem.